exercises/queen-attack/example.js

```diff
@@ -1,12 +1,15 @@
 'use strict';
 
+
 module.exports = function (options) {
-  if (options === undefined) {
-    options = { white: [0, 3], black: [7, 3] };
+  if (options === 'undefined') {
```

Comment:
this should read:
```javascript
if (options === undefined) {
```

`undefined` is a [primitive value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) in JavaScript.  You don't want to compare to the string `"undefined"`, but rather to the JS primitive which means "this variable has not been assigned any value at all."

At a high level, what you're trying to do is say: "if `options` has not been assigned a value, assign it these values."  So, line 5 should check to see if `options` has not been assigned a value, and if so, it should assign a value.  **However**, you cannot set a property on an object that has not been defined.  Unfortunately, that is what you're trying to do in lines 6 and 7.

You first need to _define_ `options` if it is `undefined`. EG:
```javascript
options = {}
```
Then you can set its properties `white` and `black`.

Even better, you can both do steps at once, like this:
```javascript
options = {
  black: [7, 3],
  white: [0, 3]
}
```